api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/pac4j/saml/Pac4jSamlClientProperties.java

```diff
@@ -289,6 +289,15 @@ public class Pac4jSamlClientProperties extends Pac4jBaseClientProperties impleme
      */
     private String messageStoreFactory = "org.pac4j.saml.store.EmptyStoreFactory";
 
+    /**
+     * Controls the way SAML attributes are converted from the SAML authentication response into pac4j attributes.
+     * <ul>
+     *     <li>{@code org.pac4j.saml.profile.converter.SimpleSAML2AttributeConverter}: Values of complex types are serialized into a single attribute. (default)</li>
```

Comment:
You should either turn these into @link and make sure the javadoc is valid, or simply remove the impls and refer to the actual Converter interface. This is not maintainable in this form specially if the impls move way or get renamed, etc, we will not be available to catch them. A link to pac4j documentation page would also be appropriate.